### PR TITLE
Goods in ut 3

### PR DESCRIPTION
**Pull request by lewisrenfrew**

### CODE REVIEW

File: src/Domain.LinnApps/GoodsIn/GoodsInService.cs [hunk 1]
```diff
@@ -136,5 +197,23 @@ public class GoodsInService : IGoodsInService
 
             return result;
         }
+
+        public ProcessResult ValidatePurchaseOrderQty(
+            int orderNumber, 
+            int qty,
+            int? orderLine)
+        {
+            var valid = this.storesPack.ValidOrderQty(orderNumber, orderLine ?? 1, qty, out _, out _);
```
Comment (iaincrawf) [inline src/Domain.LinnApps/GoodsIn/GoodsInService.cs:206]: What does out _ do?

File: src/Service.Host/client/src/components/goodsIn/GoodsInUtility.js [hunk 2]
```diff
@@ -94,6 +114,15 @@ function GoodsInUtility({
                         value={formData.qty}
                         label="Qty"
                         propertyName="qty"
+                        textFieldProps={{
+                            onBlur: () =>
+                                validatePurchaseOrderBookInQty(
+                                    `orderNumber=${formData.purchaseOrderNumber}&qty=${
+                                        formData.qty
+                                    }&orderLine=${1}&`,
```
Comment (iaincrawf) [inline src/Service.Host/client/src/components/goodsIn/GoodsInUtility.js:122]: not sure if more params added elsewhere or if a wee extra '&'

**Review by iaincrawf — APPROVED** [review]
Looks good to me

File: src/Domain.LinnApps/GoodsIn/GoodsInService.cs (hunk 1 above)
Comment (lewisrenfrew) [reply src/Domain.LinnApps/GoodsIn/GoodsInService.cs:206]: Nothing really. I think its just a way of denoting that a variable will be unused. I would guess under the hood it stops the runtime allocating memory for that variable

**Pull request merged**